docs/conflicts.md

```diff
@@ -47,3 +47,62 @@ The deeper understanding of conflicts has many advantages:
 
 For information about how conflicts are handled in the working copy, see
 [here](working-copy.md#conflicts).
+
+
+## Conflict markers
+
+Conflicts are "materialized" using *conflict markers* in various contexts. For
+example, when you run `jj edit` on a commit with a conflict, it will be
+materialized in the working copy. Conflicts are also materialized when they are
+part of a diff output (e.g. `jj show` on a commit that introduces or resolves a
+conflict). Jujutsu uses a different conflict marker style than Git uses. Here's
+an example:
+
+```
+<<<<<<<
+-------
```

Comment:
I think it's still worth identifying that the minus/plus sequence is just to indicate the start of a conflict, and there's never (?) meaningful content inside of it.